schema.sql

```diff
@@ -22,6 +22,15 @@ CREATE TABLE species(
    name VARCHAR(100)
    );
    
```

Comment:
Please add a new join table named `specializations` with the `species_id` and `vet_id` columns.
Please add a new join table named `visits` with `animal_id`, `vet_id` and `visit_date`.

Reply:
Ooh i forgot to add them. Done.